package.json

```diff
@@ -6,9 +6,10 @@
   "version": "10.2.0-pre",
   "repository": "github:grafana/grafana",
   "scripts": {
-    "build": "yarn i18n:compile && NODE_ENV=production webpack --progress --config scripts/webpack/webpack.prod.js",
+    "prebuild": "yarn i18n:compile && yarn plugin:build",
+    "build": "yarn prebuild && NODE_ENV=production webpack --progress --config scripts/webpack/webpack.prod.js",
```

Comment:
I can see the convenience of this but I wonder if it's such a big step to ask devs to run the `plugin` commands separately? 🤔 

My reasoning is that the grafana webpack build is likely depending on the output from the i18n compilation. Where as the plugins frontend assets aren't required by the webpack build to run.

Reply:
IMO it's good that `yarn build` builds _everything_ needed to run Grafana.

Devs can miss / forget the memo and it can be annoying to debug why a core plugin has suddenly stopped working.